etc/config/c++.amazon.properties

```diff
@@ -512,12 +512,14 @@ libs.fmt.versions.400.version=4.0.0
 libs.fmt.versions.400.path=/opt/compiler-explorer/libs/fmt/4.0.0
 libs.hfsm.name=HFSM
 libs.hfsm.description=Hierarchical Finite State Machine Framework
-libs.hfsm.versions=trunk:08
+libs.hfsm.versions=trunk:08:010
 libs.hfsm.url=https://github.com/andrew-gresyk/HFSM
 libs.hfsm.versions.trunk.version=trunk
-libs.hfsm.versions.trunk.path=/opt/compiler-explorer/libs/hfsm/trunk
```

Comment:
If we originally had `/opt/compiler-explorer/libs/hfsm/trunk` there might be links out there that use the library, having manually added `"include/..` to the include path, and this chang would break those.

Fortunately, we have had to deal with this kind of problem before, so it's easily fixed. Change this line to 
`libs.hfsm.versions.trunk.path=/opt/compiler-explorer/libs/hfsm/trunk:/opt/compiler-explorer/libs/hfsm/trunk/include` which will add both, new and old, paths to the include dirs. :)

Reply:
For some reason extra 'include' is needed for trunk version.
You can try this yourself in the current build:

For trunk this works:
#include <include/hfsm/machine.hpp>

For 0.8 this does:
#include <hfsm/machine.hpp>